### ISSUE TITLE
overall errors with v 0.1.0

### ISSUE THREAD

**Issue by Joduai** [pytesNET/snicker#9]
Didn't test the release version much, but not to shred between many things I launch an issue about all errors caught in initial v 0.1.0

1. captcha has its own thread

2. after setting "comments per page" to 0 to show all comments on page I get below errors
```
Warning: Division by zero in /bl-plugins/comments/system/class.snicker.php on line 346
Warning: count(): Parameter must be an array or an object that implements Countable in /bl-plugins/comments/system/class.snicker.php on line 356
Warning: Division by zero in /bl-plugins/comments/themes/default/snicker.php on line 144
Previous Comments Next Comments
Warning: Division by zero in /bl-plugins/comments/themes/default/snicker.php on line 144
```

3. using: comment avatar -> mystery men or identicon still allows to choose from "comment gravatar" option dropdown.

**Comment by SamBrishes** [comment]
Hellow,

thanks for mention.

2. I'll fix this issue as fast as possible.

3. Yes, that's intended. I don't want to chain options or create options which controls the visibility / operability of other options. For myself I think it is really disturbing if one options loads to "popup" another one. 
But I guess I can live when the option disables or enables the other one. I'll add this in the next update.

Sincerely,
Sam.

**Comment by SamBrishes** [comment]
Hellow,

this error could be fixed in version 0.1.1, and The Gravtar field doesn't get hidden, but it gets disabled to show off, that this option doesn't has any affect anymore.

Thanks.

Sincerely,
Sam,
